OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -37,6 +37,20 @@ public virtual ISpriteSequence CreateSequence(ModData modData, TileSet tileSet,
 				nodes.Remove("Defaults");
 				nodes = nodes.ToDictionary(kv => kv.Key, kv => MiniYaml.MergeStrict(kv.Value, defaults));
 
+				// Merge default animation image value. An example follows.
+				// Note: This example isn't entirely true as we remove the 'Defaults' section above.
```

Comment:
I would prefer it if we could use an example that was entirely true.  Could you fix the after case to reflect what it actually does?